UK/Data/Plugin/vSMR/aircraft-data.csv

```diff
@@ -78,7 +78,7 @@ B741,195.7,231.8,40.7
 B742,195.7,231.8,40.7
 B743,195.7,231.8,40.7
 B744,195.7,231.9,40.7
-B748,134.8,250.2,41.8
+B748,195.7,231.9,40.7
```

Comment:
Possible to adjust to actual wingspan/length?

Reply:
It should've been already but apparently not - Oliver has some updated numbers which he's going to push here